### PR TITLE
t7

### PR DESCRIPTION
**Pull request by starlingjon**
description of changes etc

### CODE REVIEW

File: fileForTicket7.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+more new stuff
+
+new lines new lines new lines of code
+code, more code
+or text, or whatever
+
+more lines
+of
+
+code
```
Comment (jonmackenzie) [inline fileForTicket7.txt:14]: This comment is about multiple lines of the submitted code
```suggestion
new lines new lines new lines of code

here's some code, more code
or text, or whatever

more lines

of

code
```

**Review by jonmackenzie — CHANGES_REQUESTED** [review]
here's a CR comment

File: fileForTicket7.txt (hunk 1 above)
Comment (starlingjon) [reply fileForTicket7.txt:14]: here's a comment asking for clarification or something

Comment (jonmackenzie) [reply fileForTicket7.txt:14]: here's a comment provided requested clarification

File: anotherFile.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+this file
+
+was added
+
+in t7
+
+after the first round
+
+of CR
```
Comment (jonmackenzie) [inline anotherFile.txt:9]: make this one line
```suggestion
after the first round of CR
```

File: fileForTicket7.txt [hunk 3]
```diff
@@ -4,14 +4,15 @@ with lots of new stuff
...
-more lines
-of
+are a complete rewrite
 
-code
+of an entire section
+
+of this file
+
+here's the end of the rewrite, this line here
```
Comment (jonmackenzie) [inline fileForTicket7.txt:15]:
```suggestion
here's the end of the rewrite:

this line here. it's the last from the rewrite
```

**Review by jonmackenzie — CHANGES_REQUESTED** [review]
two suggestions

File: fileForTicket7.txt (hunk 1 above)
Comment (jonmackenzie) [reply fileForTicket7.txt]: closing this conversation as the changes have become outdated

**Review by jonmackenzie — APPROVED** [review]
LGTM

**Pull request merged**